l10n.js

```diff
@@ -1,74 +1,17 @@
 var l10n = {
   "en": {
-    "language":"English",
```

Comment:
Uhh, why remove the perfectly fine English? You just need to change the 'unique' part of the description, not the whole thing.